### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by rsundar**
# FEATURES ADDED
- Added the required linter files for the github actions
- Added the required components for the app to work along with the necessary images.

### CODE REVIEW

File: src/contact.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        <p>For party orders or use of the hall please send us an email</p>
+        <p>party@bangaloretiffincenter.com</p>
+        <br>
+        <p>Phone Number</p>
+        <p>+91-9112764532</p>
+        <br>
+        <p>Address</p>
+        <p>Number 10, Shri Kala Building, Lavelle Road, Bengaluru-560012</p>
+      </div>
+    </div>`;
```
Comment (PhillipUg) [inline src/contact.js:20]: - [ ] Prioritize the use of `document.createElement` and `document.appendChild` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/append_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page)).
- Replace all the rest

File: src/menu.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+function setImages() {
+  const images = collectImages();
+  images.forEach(image => {
+    const index = images.indexOf(image);
+    const placeholder = document.querySelector(`#dish-${index}`);
+    placeholder.appendChild(image);
+  });
+}
```
Comment (PhillipUg) [inline src/menu.js]: - [ ] Use ES6 syntax, especially the usage of arrow functions instead of the `function` keyword.
- Replace all the rest

**Review by PhillipUg — CHANGES_REQUESTED** [review]
Hey team, 👋

You have done an amazing job implementing the requirements for this project.

However, there are still some changes 👇 you need to implement to approve this request 🔍

### Changes Required ♻️ 

- [x] First of all, you have some linter errors. Always ensure you have fixed all linter errors before submitting them for review in all your future projects.
- [x] The Live link you provided in your readme file is broken
- [x] I have followed the instructions in your readme but I am unable to run your project locally. When I open index.html, I see a blank screen
- [x] Each `module` exports a function that creates a `<div>` element, adds the appropriate content and styles to that element and then appends it to the DOM. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/tab_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page)). Prioritize the use of `document.createElement` and `document.appendChild` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/append_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page)).
- [x] Necessary run common commands like `build`, `development` should be in `package.json` inside \`scripts'. Here is a basic example ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/))
```json
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

- [x] It uses ES6 syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/)) especially usage of arrow functions instead of the `function` keyword.

#### Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack as **Phillip Musiime** or
 <a href="mailto:phillipmusiime@gmail.com"> gmail</a> &nbsp; &nbsp; [Twitter Follow](https://twitter.com/intent/follow?screen_name=Phillip_Ug) &nbsp; &nbsp; [GitHub followers](https://github.com/PhillipUg)

**Review by NtwaliHeritier — CHANGES_REQUESTED** [review]
### **STATUS: CHANGES REQUESTED🛑🛑🛑🛑**

Hello team, great work on the project. I love the effort you put in👌

[REQUESTED]: Kindly fix your linter errors
[REQUESTED]: As rules of git flow, development branch should be the default. Kindly correct it in the repo settings
[REQUESTED]: In your menu.js, kindly use arrow functions instead of function keywords, in order to use ES6.
[REQUESTED]: As mentioned by the previous TSE, you should be able to run something like npm start to start the app, not needing to go in the dist folder to manually run it.
[REQUESTED]: The demo is not working, kindly deploy it to github pages or netlify.

Kindly make the changes and request for another review👍👌
[OPTIONAL]: The links to different parts of the page can use another color as it is not visible.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @rsundar   :clap: 

**Great work making `almost` all the necessary corrections from the previous review.**

# Status: Changes Required! ♻️ 

You must have forgotten the changes mentioned by both the previous reviewers which is adding necessary scripts to run your application.
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```


### ♻️ Kindly re-submit for review after making this changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Review by mahmoud717 — CHANGES_REQUESTED** [review]
### Changes required ♻️ 

**Hello @rsundar**

Great job with the project so far it looks amazing, but there are still some changes to be made:
 
---
### Requested 🔴 

- [ ] when I open a tab other than the home one and try to go back to the home tab an error occurs. please make sure to fix that error.

### Optional 🟢 

- [ ] the font color for the tap is barely seen when not selected. please make sure to fix it.

---

please make these changes and request another code review.

if you have any questions don't hesitate to contact me on slack **MahmoudMohammad**

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required  🏗️ 🔨 
****
### Hello @rsundar 👋 
You are doing a great job with this project! 👍 Great job applying the feedback you have received from myself and the other TSEs. 🎖️ 🌟 There are just a few small issues that need to be resolved before you can move on: ⬇️ 
****

- [ ] As was pointed out to you in the previous review, there is an issue when a user is on another tab and tries to go back to the home tab. The user is unable to go back to the home tab. Please fix this

### **[OPTIONAL]**

- [ ] It is difficult to read the text on the tabs due to the color on the background image. Please consider changing the color of the text so it is more readable. 

****
### Kindly make the required changes and resubmit for a new code review once you are ready 🙏


Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

**Comment by rsundar** [comment]
Sorry I submitted the link before I pushed the commit.

**Review by BrittanyBlake — APPROVED** [review]
### Status: Approved ✅ 
****
### Hi Team! 👋 
### Great job implementing the requested changes!! 🎖️ It's a 👍 from me! Go ahead and merge this pull request and keep up the good work 😄
****


 Good Luck and Happy Coding! 🍀 👩🏾‍💻

**Pull request merged**